readme.md

```diff
@@ -94,7 +113,7 @@ How is the concept of OAuth related to a valet key?
 
 Your Answer:
 ```text
-
+OAuth allows a third-party service to perform user authentication via tokens sent back and forth between the app and an OAuth provider like Twitter or Facebook. Like a valet key, these tokens are handed off to allow the OAuth provider to ensure the user request sent through the app is genuine, and to allow the app to ensure the response from the OAuth provider is genuine.
```

Comment:
:+1:  

OAuth can provide limited access to the third-party service, similar to a valet key not allowing access to trunk/glovebox.